packages/explorer-ui/tsconfig.json

```diff
@@ -2,99 +2,47 @@
   "compilerOptions": {
     "target": "es5",
     "downlevelIteration": true,
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
+    "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
     "skipLibCheck": true,
     "strict": false,
     "forceConsistentCasingInFileNames": true,
-    "noEmit": true,
+    "noEmit": false,
     "esModuleInterop": true,
     "module": "esnext",
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
     "incremental": true,
     "paths": {
-      "@assets/": [
-        "./assets"
-      ],
-      "@assets/icons/": [
-        "./assets/icons"
-      ],
-      "@assets/*": [
-        "./assets/*"
-      ],
-      "@constants/": [
-        "./constants"
-      ],
-      "@constants/chains/": [
-        "./constants/chains"
-      ],
-      "@constants/*": [
-        "./constants/*"
-      ],
-      "@components": [
-        "./components"
-      ],
-      "@components/*": [
-        "./components/*"
-      ],
-      "@tw": [
-        "./components/tailwind"
-      ],
-      "@tw/*": [
-        "./components/tailwind/*"
-      ],
-      "@layouts": [
-        "./components/layouts"
-      ],
-      "@layouts/*": [
-        "./components/layouts/*"
-      ],
-      "@icons": [
-        "./components/icons"
-      ],
-      "@icons/*": [
-        "./components/icons/*"
-      ],
-      "@urls": [
-        "./utils/urls"
-      ],
-      "@urls/*": [
-        "./utils/urls/*"
-      ],
-      "@utils": [
-        "./utils"
-      ],
-      "@utils/*": [
-        "./utils/*"
-      ],
-      "@graphql": [
-        "./graphql"
-      ],
-      "@graphql/*": [
-        "./graphql/*"
-      ],
-      "@styles": [
-        "./utils/styles"
-      ],
-      "@styles/*": [
-        "./utils/styles/*"
-      ]
+      "@assets/": ["./assets"],
+      // "@synapse-constants": [
```

Comment:
Do we still need this path (@synapse-constants) ?

Reply:
resolved